settings.gradle.kts

```diff
@@ -31,7 +33,12 @@ include(":sample:android-app-compose")
 include(":sample:android-mpp-app")
 include(":sample:mpp-library")
 include(":sample:mpp-library:nested-module")
-include(":sample:mpp-conditional")
-include(":sample:mpp-hierarhical")
-include(":sample:mpp-mixed")
-include(":sample:jvm-app")
\ No newline at end of file
+
+// TODO: fix `resources` linking is incompatible with Windows due path separator (`/`)
```

Comment:
can you show errors from windows? i want to fix it instead of disable samples on win

Reply:
Basically, in Windows you can't `/`, you should use `\`, but I don't know if it is possible:

```
../../../mpp-library/src/commonMain/resources
```

Error:
```
Source directory 'D:\Repos\Kotlin\moko-resources\sample\mpp-mixed\src\commonMain\resources' is not a directory.
```

Reply:
I removed this one, but I am not unable to try things locally on Windows. Please try by yourself or enabler Actions for my user.